NewForce-Cohort-4/nutshell-frugal-badgers
# Vo stretch nav

# Description
The user should see a splash page instead of the main page content now, as well as a nav bar at the top of the page with links to each available feature of the app (events, news, etc). When the nav bar is clicked the page should only display the info for that widget and hide everything else. The nav bar should also only display when a user is logged in and be persistent while logged in.

Fixes #7 Dynamic Nav Bar

## Type of change

# Testing Instructions
Once downloaded, user should be able to run with local servers and use all nav bar functionality after logging in. Not all widgets have been added to main yet, and will need to be added to navbar functions when they are ready.

# Checklist:
